src/libraries/System.Runtime.InteropServices/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -1,11 +1,11 @@
 Compat issues with assembly System.Runtime.InteropServices:
 TypesMustExist : Type 'System.Runtime.InteropServices.AssemblyRegistrationFlags' does not exist in the reference but it does exist in the implementation.
+CannotChangeAttribute : Attribute 'System.Runtime.Versioning.UnsupportedOSPlatformAttribute' on 'System.Runtime.InteropServices.ComWrappers' changed from '[UnsupportedOSPlatformAttribute("android")]' in the implementation to '[UnsupportedOSPlatformAttribute("android")]' in the reference.
```

Comment:
> changed from '[UnsupportedOSPlatformAttribute("android")]' in the implementation to '[UnsupportedOSPlatformAttribute("android")]' in the reference

I assume this was auto-updated? That message is pretty odd/confusing to me...

Reply:
Yep - autoupdated. This is part of a known issue https://github.com/dotnet/arcade/issues/7585.